packs/st2cd/actions/tag_branch.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+GIT=`which git`
+REPO=$1
+DATE=`date +%s`
+BRANCH=$2
+OUTPUT=/tmp/git-output-$DATE
+
+cd ${REPO}
+OUT=`$GIT checkout ${BRANCH} -q && $GIT pull origin ${BRANCH} -q > ${OUTPUT}`
+if [[ $? == 0 ]]
+then
+  NEWTAG=`git describe --tags --abbrev=0 | awk -F '[ .]' '{print $1"."$2"."$3+1}'`
+  if [[ $? == 0 ]]
+  then
+    OUT=`$GIT tag -a ${NEWTAG} -m "Creating tag ${NEWTAG} for branch ${BRANCH}" && $GIT push origin ${NEWTAG} -q`
```

Comment:
Yeah, I would feel a lot better (and safer) we do that :P

Otherwise builds are not really deterministic and multiple pushes can cause us to tag the wrong revision.